mysite/settings.py

```diff
@@ -118,3 +130,15 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+LOGIN_URL = '/accounts/login/'
+LOGIN_REDIRECT_URL = 'http://localhost:8000/'
```

Comment:
would this break when running in production.  Probably best to parameterize it.

Reply:
or just make it `/`

Reply:
Yeah, after deploying on azure I wen't through and corrected the localhost references.  Perhaps though I could just use a '/' when pushing it to github, but during deployment I believe I'd still have to change it.

Perhaps though there's a standard convention of just using a '/' for this when pushing source.